src/core/presentation_definition.rs

```diff
@@ -310,19 +297,11 @@ impl PresentationDefinition {
                             // and the credential may satisfy the presentation definition.
                             .any(|value| validator.validate(value).is_ok());
 
-                        Ok(Some(is_valid))
+                        is_valid
                     }
-                    // Handle the case where the constraint field does contain a valid filter validator.
-                    Some(Err(e)) => Err(e),
-                    _ => Ok(None),
+                    _ => false,
```

Comment:
This might actually need to be `true` to handle `None` variant.

Reply:
020076a fixes.

Reply:
Yep, should be true